Source/AH/Wrappers.lua

```diff
@@ -0,0 +1,13 @@
+function Auctionator.AH.SendSearchQuery(...)
+  local args = {...}
```

Comment:
Why do we need the object wrapping and unpacking here?

Reply:
Vaargs and nested functions don't mix, directly (fixes a lua "compile" error)